cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1200,6 +1208,14 @@
     </description>
   </property>
 
+  <property>
+    <name>notification.kafka.topic.prefix</name>
+    <value>notifications</value>
+    <description>
+      Kafka topic name to publish notifications
+    </description>
+  </property>
+
```

Comment:
These properties are inserted in alphabetical order in each section. So, this one should be before ``notification.transport.system``.

I would suggest for the description:
```
Kafka topic prefix used to publish notifications